darwin/cli_functions.py

```diff
@@ -957,7 +959,7 @@ def delete_files(dataset_slug: str, files: List[str], skip_user_confirmation: bo
 
     except NotFound as e:
         _error(f"No dataset with name '{e.name}'")
-    except:
+    except Exception:
```

Comment:
Worthwhile to pass the exception to print so that customer knows more about what's going on?

Reply:
It's in unrelated code, and was changed to satisfy Mypy so, for now would prefer not to?